department-of-veterans-affairs/caseflow
# fix wrapping issue with flexbox

Resolves #8900 

### Description
used flexbox to wrap new lines to the left when resizing.

### Acceptance Criteria
- [ ] Elements wrap to be left aligned when they will not all fit on one line.
### Testing Plan
1. http://localhost:3000/queue/appeals/cf717bfc-7c37-4909-90bd-e38c82d5cc79
2. login as BVAAABSHIRE at 101
3. Select case with document ID: 173250939.1116 and docket (D)
4. you should see all these column headers: https://www.dropbox.com/s/y0kovmkiiqeoo5y/Screenshot%202019-01-24%2014.57.48.png?dl=0
5. If the attorney column is not on the first row, it should wrap to the left of a second row. you may need to resize your window to see the wrapping in action.

Screenshots:

Before: 
<img width="1168" alt="screen shot 2019-01-24 at 3 25 00 pm" src="https://user-images.githubusercontent.com/46791771/51706361-3e54d280-1fec-11e9-94a3-2d796805107a.png">

After:
<img width="1141" alt="screen shot 2019-01-24 at 3 25 29 pm" src="https://user-images.githubusercontent.com/46791771/51706389-4ca2ee80-1fec-11e9-99d4-6aabc42e6923.png">
